services/clipboard/index.ts

```diff
@@ -0,0 +1,11 @@
+export default function copyToClipBoard(text: string) {
+  navigator.clipboard.writeText(text).then(
+    () => {
+      alert(`copied markdown to the clipboard`);
```

Comment:
Annoying popup, I don't like it

Reply:
You can just change it to how you like it, I copied it from another file and forgot to change it, you were utilizing some useRef madness there :D